UPGRADE-1.4.md

```diff
@@ -42,8 +40,53 @@ for more details:
     imports:
         - { resource: "services_test.yaml" }
     ```
-* If you use our Travis CI configuration, follow [theses changes](https://github.com/Sylius/Sylius-Standard/pull/323/files#diff-354f30a63fb0907d4ad57269548329e3) introduced in `.travis.yml` file
+* If you use our Travis CI configuration, follow [these](https://github.com/Sylius/Sylius-Standard/pull/322/files#diff-354f30a63fb0907d4ad57269548329e3)
```

Comment:
Could we create a dummy PR on Sylius-Standard with 1.3...1.4 diff, close & lock it and then use it for diffs? It would require one link then.

Reply:
👍 but I would also vote to do it in a separate PR, to have the whole UPGRADE guide already written (as it's working) and just improve it, wdyt?